packages/dashboard/src/customization/widgets/symbolLibrary/plugin.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import RectangleWidgetComponent from './rectangle/component';
+import RectangleIcon from './rectangle/icon';
+import type { DashboardPlugin } from '~/customization/api';
+import { LineWidget, ShapeWidget } from '../types';
+import TriangleWidgetComponent from './triangle/component';
+import TriangleIcon from './triangle/icon';
+import LineWidgetComponent from './line/component';
+import LineSymbolIcon from './line/icon';
+
+export const symbolLibraryPlugin: DashboardPlugin = {
+  install: ({ registerWidget }) => {
+    registerWidget<LineWidget>('line', {
+      render: (widget) => <LineWidgetComponent {...widget} />,
+      componentLibrary: {
+        name: 'Line Symbol',
+        icon: LineSymbolIcon,
+      },
+      properties: () => ({
+        start: {
+          x: 25,
+          y: 200,
+        },
+        end: {
+          x: 375,
+          y: 200,
+        },
+      }),
+      initialSize: {
+        width: 400,
+        height: 400,
+      },
+    });
+
+    registerWidget<ShapeWidget>('rectangle', {
+      render: (widget) => <RectangleWidgetComponent {...widget} />,
+      componentLibrary: {
+        name: 'Rectangle',
+        icon: RectangleIcon,
+      },
+      properties: () => ({
+        borderStyle: 'solid',
```

Comment:
do we need to specify these here? I think they have defaults in the component

Reply:
removed properties for rectangle and triangle entirely